Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/jee/jsf/UserAuthorizationListener.java

```diff
@@ -98,7 +98,7 @@ public void afterPhase(PhaseEvent event) {
         if (session != null) {
             currentUser = (UserLogin) session.getAttribute(USER_INFO_SESSION_ATTRIBUTE);
         }
-
+        validateCSRFToken(event);
```

Comment:
You switch between using "csrf" and "CSRF" in a couple places in this PR; use the former everywhere (but still camel case to "Csrf" as appropriate)